ui/decredmaterial/editor.go

```diff
@@ -24,24 +25,38 @@ type Editor struct {
 	LineColor  color.NRGBA
 
 	flexWidth float32
-	//IsVisible if true, displays the paste and clear button.
+	//IsVisible if true, displays the clear button.
 	IsVisible bool
 	//IsRequired if true, displays a required field text at the buttom of the editor.
 	IsRequired bool
 	//IsTitleLabel if true makes the title label visible.
 	IsTitleLabel bool
 	//Bordered if true makes the adds a border around the editor.
 	Bordered bool
+	//IsPassword if true, displays the show and hide button.
+	IsPassword bool
 
 	requiredErrorText string
 
-	pasteBtnMaterial IconButton
-	clearBtMaterial  IconButton
+	clearBtMaterial IconButton
+
+	showHidePassword *widget.Clickable
+	iconShow         *widget.Image
+	iconhide         *widget.Image
```

Comment:
These icons are available in iconvg format. If an icon is available in iconvg format, it should be used instead of the png equivalent.